content/docs/introduction/read-replicas.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Read replicas
+subtitle: Maximize scalability and more with Neon's instant read replica feature
+enableTableOfContents: true
+isDraft: true
+---
+
+Neon read replicas are independent read-only compute instances designed to perform read operations on the same data as your read-write computes. The Neon read replica implementation does not replicate data across database instances. Instead, all read requests are directed to a single source — a capability made possible by Neon's architecture, which separates storage and compute. The following diagram shows how read requests from both read-write and read-only compute instances are served from the same Neon [Pageserver](/docs/refernce/glossary#pageserver).
+
+![Read-only compute instances](/docs/introduction/read_replicas.png)
+
+In data replication terms, Neon's read replica feature is considered asynchronous. As updates are made by your read-write computes, the latest changes are made available to Neon Pageservers via a Write-Ahead Log (WAL) stream. Neon's read replicas offer near-instant data consistency for read replicas residing in the same region as your database. Cross-region read replicas are currently not supported. You can expect that feature in a future release.
+
+You can instantly create one or more read replicas for any branch in your Neon project and configure the amount of vCPU and memory allocated to each. Read replicas also support Neon's Autoscaling and Auto-suspend features, providing you with control over the compute resources used by your read replicas.
+
+## Use cases
+
+Neon's read replicas have a number of potential applications:
+
+- **Throughput enhancement**: By distributing read requests among multiple read replicas, you can achieve higher throughput for both read-write and read-only workloads.
+- **Workload offloading**: Assign reporting or analytical workloads to a read replica to prevent any impact on the performance of read-write application workloads.
+- **Access control**: Provide read-only data access to certain users or applications that do not need write access.
+- **Resource customization**: Configure right-sized CPU and memory resources to cater to the specific needs of different users and applications.
+
+## Advantages
+
+Advantages of Neon's read replica feature include the following:
+
+1. **Efficient storage**: With read-only compute instances reading from the same source as your read-write computes, no additional storage is required to create a read replica. Data is neither duplicated nor replicated.
+2. **Data consistency**: Read-write and read-only compute instances read data from a single source, ensuring the highest degree of data consistency.
```

Comment:
https://neondb.slack.com/archives/C04345PBJ49/p1685087949214039
look at these things too. Maybe it'll help to add more benefits

Reply:
Thank you. I will take a look to see what I can add.

Reply:
I think I have worked in these ideas now.